site/layouts/partials/docs-sidebar.html

```diff
@@ -1,4 +1,4 @@
-<nav class="bd-links w-100" id="bd-docs-nav" aria-label="Docs navigation">
+<nav class="bd-links w-100 pb-2" id="bd-docs-nav" aria-label="Docs navigation">
```

Comment:
We don't have it in Boosted or Bootstrap, do we absolutely need it here?

Reply:
It feels better to me since the same padding is added at the top, but you're right it should be done Bootstrap side, I'll try to gather all the ideas raised in the OUDS Web PRs to make them happen on Bootstrap side.